public/js/term-highlight.js

```diff
@@ -0,0 +1,34 @@
+jQuery(function($) {
+  $(document).ready(function () {
+    var searchTerm = $('h1.sr-only').data('term').toLowerCase();
+        // remove speach mark
+        searchTerm = searchTerm.replace(/[".]/g,'');
+        // remove special characters
+        searchTerm = searchTerm.replace(/[;]/g,'');
+        // Change Doctor to Dr
+        searchTerm = searchTerm.replace('doctor','dr');
+
+    var terms = searchTerm.split(/\W+/);
+
+    for (var i = 0, len = terms.length; i < len-1; i++) {
+      terms.push(terms[i]+' '+terms[i+1]);
+    }
+
+    if ($.inArray( searchTerm, terms ) === -1){
```

Comment:
I'd prefer to see the use of native JS code, however, I'm aware functions like [`.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes?v=control) isn't supported in IE (according to those docs) and [`.indexOf()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf?v=control) is only supported in IE9+ 🤕